src/content/Angular-CSharp/.template.config/localize/templatestrings.en.json

```diff
@@ -17,7 +17,8 @@
   "symbols/NoHttps/description": "Whether to turn off HTTPS. This option only applies if Individual, IndividualB2C, SingleOrg, or MultiOrg aren't used for --auth.",
   "symbols/UseLocalDB/description": "Whether to use LocalDB instead of SQLite. This option only applies if --auth Individual or --auth IndividualB2C is specified.",
   "symbols/Framework/description": "The target framework for the project.",
-  "symbols/Framework/choices/net7.0/description": "Target net7.0",
+  "symbols/Framework/choices/net8.0/description": "Target net8.0",
+  "_symbols/Framework/choices/net8.0/description.comment": "{Locked='net8.0'}",
```

Comment:
@wtgodbe this is to avoid messes like the translations in dotnet/aspnetcore#45630. We need the same thing in all templatestrings.en.json files in dotnet/aspnetcore

Reply:
What exactly does this do? Would it prevent changing any part of that string, or is it only intended to protect that lowercase n?

Reply:
I'm asking how it works mechanically - does `net8.0` correspond exactly to the `net8.0` portion of `Target net8.0` in the line above? Will it only affect translations of this template, or future updates to `net9.0` as well?

Reply:
> I'm asking how it works mechanically - does `net8.0` correspond exactly to the `net8.0` portion of `Target net8.0` in the line above? Will it only affect translations of this template, or future updates to `net9.0` as well?

These comments are documented somewhere but the gist is that a `{Locked}` comment prevents translation of the entire string while `{Locked='net8.0'}` just locks 'net8.0', leaving `Target` to be translated.

We should update the loc comment at the same time we change the string for `net9.0`.